### ISSUE TITLE
How can I classify 200,000 categories? use softmax loss .

### ISSUE THREAD

**Issue by xf739645524** [GNAYUOHZ/ReID-MGN#13]

**Comment by GNAYUOHZ** [comment]
If I understand correctly，change num_classes in network.py to 200,000. 
Maybe I should write it in argparse.

**Comment by xf739645524** [comment]
Okey ，
I trained for a week and the crossentropy loss finally came down.

**Comment by guanyonglai** [comment]
Hi, how about the results when you train 20W IDs? Better than less IDs?

**Comment by xf739645524** [comment]
> Hi, how about the results when you train 20W IDs? Better than less IDs?

This loss is very difficult to drop

**Comment by guanyonglai** [comment]
> This loss is very difficult to drop

I have trained on my own dataset, The 8000ID training produced the best results,If you add more ID, the accuracy starts to go down.Have you solved this problem yet?
